### ISSUE TITLE
quotemark single and newline character

### ISSUE THREAD

**Issue by olegstepura** [palantir/tslint#1313]
### Bug Report

- __TSLint version__: `3.10.2`
- __TypeScript version__: `1.8.10`
- __Running TSLint via__:  webpack

#### TypeScript code being linted

```ts
testString.split(' <- ').join("\n")
```

with `tslint.json` configuration:

```json
{ "quotemark": [true, "single", "jsx-double", "avoid-escape"] }
```

#### Actual behavior

```
WARNING in ...: " should be '
```

#### Expected behavior

Newline character can only be entered in double quotes, so I suggest either silently allowing it via `"avoid-escape"` or ignore completely if  `"single"` rule is used for `"quotemark"`

**Comment by olegstepura** [comment]
Hmm, seems like `'\n'` works fine, so there is no need to use double quotes here. In my case it was `React`, that made me feel like this does not work (rendered text was containing `\n` chars instead of newline or at least space). Closing this.
